test/test-coordinator.js

```diff
@@ -245,9 +245,13 @@ TestCoordinator.prototype.callEndpoint = function callEndpoint(endpoint, body, c
 };
 
 TestCoordinator.prototype.getMembership = function getMembership() {
+    var sutHost = this.sutHostPort.split(':')[0]
+    var sutPort = this.sutHostPort.split(':')[1]
+
     return this.getFakeNodes().map(function(node) {
         return node.toMemberInfo();
-    }).concat(this.extraMembers);
+    }).concat(this.extraMembers)
+    .concat({host:sutHost, port:sutPort, status:'alive', incarnationNumber:this.sutIncarnationNumber});
```

Comment:
I find the SUT membership information to be very useful -- maybe you can put this dictionary to a function in `tc`?